package.json

```diff
@@ -3,10 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "develop": "npm-run-all dev",
-    "dev": "next dev",
```

Comment:
I really like the dev shortcut, please don't take it away from me :pray:

Reply:
Ahahah, I'll fix it in the next commit :+1: